app/services/katello/toggle_simple_content_access.rb

```diff
@@ -0,0 +1,47 @@
+module Katello
+  class ToggleSimpleContentAccess
```

Comment:
This is a good abstraction, but it would be more appropriate I think to create an 'Action' class that we can execute from the controller, which itself will 'plan' the manifest refresh.

I think it could live at `app/lib/actions/katello/organization/simple_content_acces/{enable,disable}.rb` (two distinct actions)